pkgs/development/interpreters/python/build-python-package-common.nix

```
@@ -4,10 +4,13 @@
 }:
 
 { buildInputs ? []
-# Additional flags to pass to "pip install".
+# `install-options` to pass to `pip install`.
 , installFlags ? []
 , ... } @ attrs:
 
+let
+  installFlagsString = lib.concatMapStringsSep " " (option: "--install-option ${option}") installFlags;
+
```

I'm able to build python packages again after these two changes
```suggestion
in
```